.github/dependabot.yml

```
@@ -0,0 +1,13 @@
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "weekly"
+  - package-ecosystem: "pip"
+    directory: "/tests"
```

Dependabot [now supports](https://github.blog/changelog/2023-06-30-grouped-version-updates-for-dependabot-public-beta/) grouped version updates where it files one PR to update the entire lockfile like we do in the ansible-documentation repo instead of a bunch of individual PRs that create a lot of noise. That might be preferable here.

```suggestion
    directory: "/tests"
    groups:
      dependencies:
        patterns:
          - "*"
```